test/clients/rtm/client.js

```diff
@@ -140,6 +140,27 @@ describe('RTM API Client', function () {
 
   });
 
+  describe('Message Updating', function () {
+
+    it('should call a cb with an err when the RTM is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
+      rtm.updateMessage({}, function (err, res) {
+        expect(err).to.not.equal(null);
+        expect(res).to.equal(null);
+        done();
+      });
+    });
+
+    it('should call a catch cb with an err when the RTM client is not connected', function (done) {
+      var rtm = new RtmAPIClient('fake-token');
```

Comment:
Probably the thing to do—at least what I do with the client I develop on my own—is simply require a valid token in the environment. And then provide one to a test team you've set up for the purposes of running tests.